MeetupPlatformApi/Features/Authentication/RegisterNewUser/RegisterNewUserFeature.cs

```diff
@@ -24,24 +25,24 @@ public RegisterNewUserFeature(ApplicationContext context, IMapper mapper, TokenH
     }
 
     /// <summary>
-    /// Register new user.
+    /// Register new organizer.
     /// </summary>
     /// <response code="201">Returns registration result data.</response>
     /// <response code="400">If provided username is already taken.</response>
-    [HttpPost("/api/users")]
+    [HttpPost("/api/users/organizer")]
     [ProducesResponseType(typeof(RegistrationResultDto), StatusCodes.Status201Created)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
-    public async Task<IActionResult> RegisterNewUser([FromBody] RegistrationDto registrationDto)
+    public async Task<IActionResult> RegisterNewOrganizer([FromBody] RegistrationDto registrationDto)
```

Comment:
It seems to me that there should be some kind of check so that any user does not register as an organizer